spec/javascripts/test_spec.js

```diff
@@ -57,6 +57,27 @@ describe("Basic testing", function() {
     blurredLocation.setBlurred(false);
     expect(blurredLocation.isBlurred()).toBe(false);
   });
+
+  it("Checks if input listeners change maps position to the entered latitude and longitude", function() {
+
+    var lat = document.getElementById("lat");
+    var lng = document.getElementById("lng");
+
+    lat.value = 20;
```

Comment:
Hm, not sure about the type error, sorry i've been reading a lot of code so what do you refer to?


Actually I think you need to be using the jquery mode of getting the element, like `$('idName')` rather than `getElementById` - the latter doesn't have the extra features of a jQuery element, like the `val()` method. So:

`var lng = document.getElementById("lng");`

becomes

`var lng = $("lng");`

then `lngEl.val(20)` should work.